pkg/commands/backend/create.go

```diff
@@ -67,30 +77,30 @@ func NewCreateCommand(parent cmd.Registerer, globals *config.Data, data manifest
 		Action: c.autoClone.Set,
 		Dst:    &c.autoClone.Value,
 	})
-	c.CmdClause.Flag("name", "Backend name").Short('n').Required().StringVar(&c.input.Name)
-	c.CmdClause.Flag("address", "A hostname, IPv4, or IPv6 address for the backend").Required().StringVar(&c.input.Address)
-	c.CmdClause.Flag("comment", "A descriptive note").StringVar(&c.input.Comment)
-	c.CmdClause.Flag("port", "Port number of the address").Action(c.port.Set).UintVar(&c.port.Value)
+	c.CmdClause.Flag("name", "Backend name").Short('n').Action(c.name.Set).StringVar(&c.name.Value)
```

Comment:
Not essential, but I'd appreciate it if it was done as part of this PR:
Could we make sure the flags in each command are sorted in ascending order (just makes it easier when looking at the --help output to find flags).

Reply:
Ok, all sorted in https://github.com/fastly/cli/pull/707/commits/cbe12eebe989695bad94f383abb85726e0433f89